ios/FluentUI.xcodeproj/project.pbxproj

```diff
@@ -1726,6 +1726,7 @@
 			baseConfigurationReference = 53FC90F525673626008A06FD /* FluentUI_release.xcconfig */;
 			buildSettings = {
 				CLANG_WARN_QUOTED_INCLUDE_IN_FRAMEWORK_HEADER = YES;
+				IPHONEOS_DEPLOYMENT_TARGET = 15.0;
```

Comment:
these changes required?

Reply:
Looks like it'll just pull it from the xcconfig, so we're fine without these.